b/byte-buddy-byte-buddy/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "santhoshkotla2929",
+    "package_name": "byte-buddy-byte-buddy",
```

Comment:
Package name can be just byte-buddy. Update wherever applicable - folder name, script header, build info.